packages/suite/src/components/suite/modals/TransactionDetail/components/ChainedTxs/index.tsx

```diff
@@ -16,12 +16,16 @@ const StyledTrezorLink = styled(TrezorLink)`
 
 interface Props {
     txs: WalletAccountTransaction[];
+    explorerUrl?: string;
 }
 
-const ChainedTxs = ({ txs }: Props) => (
+const ChainedTxs = ({ txs, explorerUrl }: Props) => (
     <Wrapper>
         {txs.map(tx => (
-            <StyledTrezorLink href={getBlockExplorerUrl(tx)} variant="nostyle">
+            <StyledTrezorLink
+                href={explorerUrl ? `${explorerUrl}/tx/${tx.txid}` : getBlockExplorerUrl(tx)}
```

Comment:
you can do this check only once: in `TransactionDetail/index` and pass it here and into `BasicDetails` as required string

Reply:
fixed in 2ea85cd